src/pages/authorization/customer-authorization/index.mdx

```diff
@@ -136,30 +137,33 @@ Agent token grant is recommended for backend integrations. It allows you to impe
-| `grant_type` | yes      | Use `agent_token` for this authorization flow.                                                                                                                                             |
-| `client_id`  | yes      | Client ID of your application                                                                                                                                                              |
-| `entity_id`  | no       | Customer ID. If you don't specify it, new customer identity will be created. Specify it if you want to [acquire a new token for an existing customer identity](#case-existing-customer-1). |
+| Parameter         | Required | Description                                                                                                                                                                                                                                                                                                                             |
+| ----------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `grant_type`      | yes      | Use `access_token` (agent access token) for this authorization flow.                                                                                                                                                                                                                                                                    |
+| `client_id`       | yes      | Client ID of your application                                                                                                                                                                                                                                                                                                           |
+| `response_type`   | yes      | Value: `token`                                                                                                                                                                                                                                                                                                                          |
+| `entity_id`       | no       | Customer ID. If you don't specify it, a new customer identity will be created ([Case: New customer](#case-existing-customer-1)). Specify it if you want to [acquire a new token for an existing customer identity](/getting-started/authorization/faq/#the-access-token-of-my-customer-has-expired-do-i-need-to-create-a-new-customer). |
```

Comment:
I am sorry I have not seen it before, but is the link to the `Case: New customer` correct? Shouldn't it link [here](https://developers.livechat.com/docs/authorization/customer-authorization#case-new-customer-1)?